federation/cluster/federation-up.sh

```diff
@@ -90,7 +90,7 @@ function init() {
       --dns-zone-name="${DNS_ZONE_NAME}" \
       --dns-provider="${DNS_PROVIDER}" \
       --image="${kube_registry}/hyperkube-amd64:${kube_version}" \
-      --apiserver-arg-overrides="--storage-backend=etcd2" \
+      --apiserver-arg-overrides="--storage-backend=etcd2,--runtime-config=api/all=true" \
```

Comment:
> This will have the effect of enabling all APIs for any federated bringup where environment in the case that USE_KUBEFED=1.
> 
> I suggest that you add an environment variable APISERVER_ARG_OVERRIDES to federation-up.sh which gets applied in both USE_KUBEFED cases. Then on the e2e side, we can freely set this parameter by configuring jenkins job environments.

As far as I understand, the federation-up.sh script is used only to setup the federation e2e env (https://github.com/kubernetes/kubernetes/blob/master/federation/cluster/federation-up.sh#L21).
Also the test-infra job too has only one profile for federation e2e which always calls this script with a particular set of flags which always enables the ```USE_KUBEFED=1```. I don't see any specific need of creating an additional flag, thus. 
Probably @madhusudancs might be able to suggest something better.